src/core_plugins/kbn_vislib_vis_types/public/editors/heatmap.html

```diff
@@ -0,0 +1,47 @@
+<div class="kuiSideBarSection">
+  <div class="kuiSideBarSectionTitle">
+    <div class="kuiSideBarSectionTitle__text">
+      Global Settings
```

Comment:
Hmm do you maybe have any suggestion for the title ? just Options ? Basic Options ?